bin/fetch-breaches.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const path = require('path');
+
+const fetch = require('node-fetch');
+const { stripIndents } = require('common-tags');
+
+const API_BASE_URL = 'https://haveibeenpwned.com/api/v2';
+
+function getBreaches() {
+  return fetch(`${API_BASE_URL}/breaches`)
+    .then(res => res.json());
+}
+
+getBreaches()
+  .then(breaches => breaches.map(({ BreachDate, Domain }) => `${Domain} (${BreachDate})`).sort())
```

Comment:
I know we already discussed keeping the [localData.json](https://github.com/mozilla/security-advisor-shield-study/blob/master/data/recommendation/localData.json) in sync, but it'd be super trivial to dump put the breaches data right around here when we update the domains.txt file.